src/components/Radio/Radio.tsx

```diff
@@ -43,18 +43,18 @@ export const Radio = forwardRef((props: TRadioProps, ref): ReactElement | null =
         disabled={disabled}
       />
       <span className="controller__icon"></span>
-      <div className="controller__right">
-        {label ? (
+      {label ? (
```

Comment:
@RamellaWeb jan maybe someone has sent only helperText without label, and in that case that someone will not see his helperText